be/test/exec/vectorized/parquet_scanner_test.cpp

```diff
@@ -496,6 +499,12 @@ TEST_F(ParquetScannerTest, test_to_json) {
              {R"({"s0": 1, "s1": {"s2": 3}}                                                     )",
               R"({"s0": 2, "s1": {"s2": 4}}                                                     )",
               R"({ "s0": 3, "s1": {"s2": 5}}                                                    )"}},
+
+            {"col_json_struct_string",
+             {R"('{"s0": 1, "s1": "string1"}'                                                    )",
```

Comment:
Why are these string is quoted with single quote?

Reply:
This single-quoted string is the load result, and since the column type is `VARCHAR`, the json would be cased into a string with quote